src/coreclr/System.Private.CoreLib/src/System/GC.CoreCLR.cs

```diff
@@ -838,7 +838,7 @@ public static TimeSpan GetTotalPauseDuration()
             return new TimeSpan(_GetTotalPauseDuration());
         }
 
-        internal struct GCConfigurationContext
+        internal ref struct GCConfigurationContext
```

Comment:
Nit: This struct is unnecessary. The code can just pass along the address of a `Dictionary<string, object>` local.

Reply:
Ignore this comment - I see that we use this trivial wrapper structs for callbacks in other places.